src/app/footer/footer.component.ts

```diff
@@ -59,9 +48,17 @@ export class FooterComponent implements OnInit {
     return about.name;
   }
 
-  constructor() { }
+  constructor(private categoriesList: FooterService) { }
```

Comment:
Instead of categoriesList, use footerService. It is the convention.
Reason: In future, you might need to access the brand then categoriesList will not make sense.

Reply:
@akashsinha13  modified